server/migrations/1615361765942_index-optimization.js

```diff
@@ -0,0 +1,10 @@
+const { collections } = require('../db/schema.js');
+
+exports.up = async (pgm) => {
+  await pgm.createIndex(collections.PARTICIPANTS_STATUS, 'status');
+  await pgm.createIndex(collections.PARTICIPANTS_STATUS, 'employer_id');
+  await pgm.createIndex(collections.PARTICIPANTS_STATUS, 'participant_id');
+  await pgm.createIndex(collections.PARTICIPANTS_STATUS, 'current', { where: 'current = true' });
+  await pgm.createExtension('pg_trgm');
```

Comment:
What is this, exactly?

Reply:
This is necessary to enable the `gin_trgm_ops` opclass for the GIN index